landoapi/stacks.py

```diff
@@ -142,6 +143,22 @@ def __init__(self, nodes: set[str], edges: set[tuple[str, str]]):
             self.children[parent].add(child)
             self.parents[child].add(parent)
 
+    def base_revisions(self) -> Iterator[str]:
+        """Return the set of base revisions in the stack."""
```

Comment:
Would be good to add a couple of examples in the docstring giving a couple of stack configurations as examples.